routes/net-state-files.go

```diff
@@ -0,0 +1,79 @@
+package routes
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/storage/boltdb"
+)
+
+var (
+	ErrReadReq = errors.New("error reading request body")
+)
+
+type File struct {
+	DB *boltdb.Client
+}
+
+type Message struct {
+	Value string `json:"value"`
+}
+
+func NewFile(db *boltdb.Client) *File {
+	return &File{DB: db}
+}
+
+func (f *File) Put(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	givenPath := ps.ByName("path")
+
+	var msg Message
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&msg)
+	if err != nil {
+		log.Printf("err decoding response: %v", err)
+		return
+	}
+
+	file := boltdb.File{
+		Path:  givenPath,
+		Value: msg.Value,
+	}
+
+	if err := f.DB.Put(file); err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Fprintf(w, "PUT to %s\n", givenPath)
+}
+
+func (f *File) Get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	fileKey := ps.ByName("path")
+
+	fileInfo, err := f.DB.Get([]byte(fileKey))
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Fprintf(w, "value: %s", fileInfo)
+}
+
+func (f *File) List(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	fileKeys, err := f.DB.List([]byte("files"))
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Fprintf(w, "All stored file paths: %s", fileKeys)
```

Comment:
you could return json like above here too with fileKeys.

Reply:
obviously, the json doesn't matter if we switch to grpc so don't worry about any of my json comments really. just for info

Reply:
Initially I was also thinking I didn't need to worry too much about formatting to JSON if it would be switched over to gRPC, but @jhagans3 had mentioned it could be good to have both a REST/http interface and a gRPC interface (both calling the same db client methods), so I think I will do that unless you have a diff suggestion @jtolds ?

Reply:
If we want that in the future there is:
https://github.com/grpc-ecosystem/grpc-gateway